Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/WPF/InkToolbar/InkToolbar.cs

```diff
@@ -0,0 +1,192 @@
+            Bind(nameof(UseLayoutRounding), UseLayoutRoundingProperty, global::Windows.UI.Xaml.Controls.InkToolbar.UseLayoutRoundingProperty);
+            Bind(nameof(Name), NameProperty, global::Windows.UI.Xaml.Controls.InkToolbar.NameProperty);
+            Bind(nameof(Tag), TagProperty, global::Windows.UI.Xaml.Controls.InkToolbar.TagProperty);
+            Bind(nameof(DataContext), DataContextProperty, global::Windows.UI.Xaml.Controls.InkToolbar.DataContextProperty);
+            Bind(nameof(Width), WidthProperty, global::Windows.UI.Xaml.Controls.InkToolbar.WidthProperty);
+
+            // InkToolbar specific properties
+            Bind(nameof(TargetInkCanvas), TargetInkCanvasProperty, global::Windows.UI.Xaml.Controls.InkToolbar.TargetInkCanvasProperty, new WindowsXamlHostWrapperConverter());
+            Bind(nameof(IsRulerButtonChecked), IsRulerButtonCheckedProperty, global::Windows.UI.Xaml.Controls.InkToolbar.IsRulerButtonCheckedProperty);
+            Bind(nameof(InitialControls), InitialControlsProperty, global::Windows.UI.Xaml.Controls.InkToolbar.InitialControlsProperty);
+            Bind(nameof(ActiveTool), ActiveToolProperty, global::Windows.UI.Xaml.Controls.InkToolbar.ActiveToolProperty, new WindowsXamlHostWrapperConverter());
+            Bind(nameof(InkDrawingAttributes), InkDrawingAttributesProperty, global::Windows.UI.Xaml.Controls.InkToolbar.InkDrawingAttributesProperty);
+            Bind(nameof(Orientation), OrientationProperty, global::Windows.UI.Xaml.Controls.InkToolbar.OrientationProperty);
+            Bind(nameof(IsStencilButtonChecked), IsStencilButtonCheckedProperty, global::Windows.UI.Xaml.Controls.InkToolbar.IsStencilButtonCheckedProperty);
+            Bind(nameof(ButtonFlyoutPlacement), ButtonFlyoutPlacementProperty, global::Windows.UI.Xaml.Controls.InkToolbar.ButtonFlyoutPlacementProperty);
+
+            Children.OfType<WindowsXamlHost>().ToList().ForEach(RelocateChildToUwpControl);
+            UwpControl.ActiveToolChanged += OnActiveToolChanged;
+            UwpControl.EraseAllClicked += OnEraseAllClicked;
+            UwpControl.InkDrawingAttributesChanged += OnInkDrawingAttributesChanged;
+            UwpControl.IsRulerButtonCheckedChanged += OnIsRulerButtonCheckedChanged;
+            UwpControl.IsStencilButtonCheckedChanged += OnIsStencilButtonCheckedChanged;
+
+            base.OnInitialized(e);
+        }
+
+        private void RelocateChildToUwpControl(WindowsXamlHost obj)
+        {
+            VisualTreeHelper.DisconnectChildrenRecursive(obj.desktopWindowXamlSource.Content);
+            obj.desktopWindowXamlSource.Content = null;
+            Children.Remove(obj);
+            UwpControl.Children.Add(obj.XamlRoot);
+        }
+
+        public static DependencyProperty ActiveToolProperty { get; } = DependencyProperty.Register(nameof(ActiveTool), typeof(Microsoft.Toolkit.Win32.UI.Controls.WPF.InkToolbarToolButton), typeof(InkToolbar));
```

Comment:
>DependencyProperty.Register(nameof(ActiveTool), typeof(Microsoft.Toolkit.Win32.UI.Controls.WPF.InkToolbarToolButton), typeof(InkToolbar)); [](start = 71, length = 138)

Does this syntax cause the DP to get re-registered on each "get" of the property?  That wouldn't be good if so.